xmppserver/src/test/java/org/jivesoftware/util/CacheUtilTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2019 Ignite Realtime Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.util;
+
+import org.jivesoftware.util.cache.Cache;
+import org.jivesoftware.util.cache.CacheUtil;
+import org.jivesoftware.util.cache.DefaultLocalCacheStrategy;
+import org.junit.Test;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+/**
+ * Unit tests that verify the functionality of {@link CacheUtil}
+ *
+ * @author Guus der Kinderen, guus.der.kinderen@gmail.com
+ */
+public class CacheUtilTest
```

Comment:
General comment: There's no need for the try/catch. Any exception thrown that isn't an expectedException will cause the test to fail.

Reply:
Yeah ... but it fails with an error, not with an assertion failure.

Meh.